src/amqp_management.c

```diff
@@ -1101,27 +1119,56 @@ int amqp_management_close(AMQP_MANAGEMENT_HANDLE amqp_management)
     return result;
 }
 
-int amqp_management_execute_operation_async(AMQP_MANAGEMENT_HANDLE amqp_management, const char* operation, const char* type, const char* locales, MESSAGE_HANDLE message, ON_AMQP_MANAGEMENT_EXECUTE_OPERATION_COMPLETE on_execute_operation_complete, void* on_execute_operation_complete_context)
+static void amqp_management_execute_cancel_handler(ASYNC_OPERATION_HANDLE execute_operation)
 {
-    int result;
+    OPERATION_MESSAGE_INSTANCE* instance = GET_ASYNC_OPERATION_CONTEXT(OPERATION_MESSAGE_INSTANCE, execute_operation);
+
+    if (instance->send_async_context != NULL)
+    {
+        (void)async_operation_cancel(instance->send_async_context);
+    }
```

Comment:
Interesting that this has a NULL check and the other ones in cbs do not.

Reply:
Fixed.